src/subaccounts.py

```diff
@@ -0,0 +1,57 @@
+from dataclasses import dataclass
+from typing import Dict
+
+from v4_proto.dydxprotocol.subaccounts.streaming_pb2 import StreamSubaccountUpdate
+
+@dataclass(frozen=True)
+class SubaccountId:
+    owner_address: str
+    subaccount_number: int
+
+@dataclass
+class SubaccountPerpetualPosition:
+    perpetual_id: int
+    quantums: int
+
+@dataclass
+class SubaccountAssetPosition:
+    asset_id: int
+    quantums: int
+
+@dataclass
+class StreamSubaccount:
+    subaccount_id: SubaccountId
+    perpetual_positions: Dict[int, SubaccountPerpetualPosition]
```

Comment:
maybe a comment to say that the keys are perp ids. ditto below

Reply:
done